src/VisualStudio/Core/Impl/Options/AbstractSettingsManagerOptionSerializer.cs

```diff
@@ -27,16 +27,15 @@ private class SVsSettingsPersistenceManager { };
         protected readonly ISettingsManager Manager;
         private readonly IOptionService _optionService;
 
-        public AbstractSettingsManagerOptionSerializer(IServiceProvider serviceProvider, IOptionService optionService)
+        public AbstractSettingsManagerOptionSerializer(VisualStudioWorkspaceImpl workspace)
```

Comment:
right now IOptionService is a bit weird since it can be MEF service itself but also can be workspace service. probably we need to split that in to 2. ex) IOptionService to be workspace specific and create IAggregatedOptionService or something that can change options for all workspaces or find out value of specific options for all workspace and etc.